.github/workflows/ci-app-examples.yml

```diff
@@ -72,10 +72,12 @@ jobs:
           key: ${{ runner.os }}-pip-py${{ matrix.python-version }}-${{ matrix.pkg-name }}-${{ matrix.requires }}-${{ hashFiles('requirements/app/base.txt') }}
           restore-keys: ${{ runner.os }}-pip-py${{ matrix.python-version }}-${{ matrix.pkg-name }}-${{ matrix.requires }}-
 
-      - name: Install dependencies
+      - name: Install Lightning package & dependencies
+        env:
+          PACKAGE_NAME: ${{ matrix.pkg-name }}
+          FREEZE_REQUIREMENTS: 1
         run: |
-          pip --version
-          pip install -r requirements/app/devel.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+          pip install -e .[dev] --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
```suggestion
          # do not use -e because it will make both packages available since it adds `src` to `sys.path` automatically
          pip install .[dev] --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
```